### ISSUE TITLE
[css-values-5] Consider supporting `calc-size()` in the `inset` property

### ISSUE THREAD

**Issue by yisibl** [w3c/csswg-drafts#10355]
In the `inset` property we also often encounter a change from `auto` to a specific value.

If we search, we will find many similar queries:

- https://stackoverflow.com/questions/31575484/transition-to-and-from-position-auto

**Comment by tabatkins** [comment]
Yeah, the issue here is that `auto` is a behavior switch in how it interacts with its surroundings - `justify/align-self` cause it to align in its staticpos containing block instead of its abspos containing block, it considers its staticpos containing block's writing mode to be its parent writing mode, etc. We can't meaningfully compare the two.

I think the most straightforward answer is just "use anchor positioning instead of `inset: auto`".
